sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -338,8 +338,15 @@ def _read_eeprom_specific_bytes(self, sysfsfile_eeprom, offset, num_bytes):
             return None
 
         try:
-            for n in range(0, num_bytes):
-                eeprom_raw[n] = hex(ord(raw[n]))[2:].zfill(2)
+            # in case raw is bytes (python3 is used) raw[n] will return int,
+            # and in case raw is str(python2 is used) raw[n] will return str,
+            # so for python3 the are no need to call ord to convert str to int.
```

Comment:
Thanks for this fix. Can you also please add a `#TODO: Remove this check once we no longer support Python 2` comment here for posterity?

Reply:
Done